tests/test_dataset.py

```diff
@@ -0,0 +1,253 @@
+import os
+import posixpath
+
+import pytest
+
+import audeer
+import audiofile
+import audb
+import audformat
+import audbcards
+import audplot
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+BUILD = audeer.path('..', 'build', 'html')
+
+
+def test_dataset(cache, tmpdir, db):
+    dataset = audbcards.Dataset(pytest.NAME, pytest.VERSION)
+
+    # __init__
+    assert dataset.name == pytest.NAME
+    assert dataset.version == pytest.VERSION
+    assert dataset.repository == pytest.REPOSITORY
+    expected_header = audb.info.header(
+        pytest.NAME,
+        version=pytest.VERSION,
+        cache_root=cache,
+    )
+    assert str(dataset.header) == str(expected_header)
+    expected_deps = audb.dependencies(
+        pytest.NAME,
+        version=pytest.VERSION,
+        cache_root=cache,
+    )
+    pd.testing.assert_frame_equal(dataset.deps(), expected_deps())
+
+    # archives
+    assert dataset.archives == str(len(db.files))
+
+    # bit depths
+    expected_bit_depths = ', '.join(set([
+        str(audiofile.bit_depth(file)) for file in db.files
+        if audiofile.bit_depth(file)
+    ]))
+    assert dataset.bit_depths == expected_bit_depths
+
+    # channels
+    expected_channels = ', '.join(set(
+        [str(audiofile.channels(file)) for file in db.files]
+    ))
+    assert dataset.channels == expected_channels
+
+    # duration
+    expected_duration = str(db.files_duration(db.files).sum())
+    assert dataset.duration == expected_duration
+
+    # example
```

Comment:
Here it would be good to add a little bit more documentation, e.g.
```suggestion
    # example
    # path to audio file from database
```
and maybe state how the `expected_example` path looks like, e.g. `path/relative/to-what/media/audio.wav` (because I don't know how it should look like ;) )